htdocs/info/about/collaborations/index.html

```diff
@@ -13,20 +13,15 @@ <h1>Collaborations</h1>
 short intensive collaborative project. 
 <a href="/info/about/outreach/geek4aweek.html">More information</a>...</p>
 
-<p>
-Formal requests for, say, large scale collaborations, legal issues, etc., are best sent to 
-<a href="http://www.ebi.ac.uk/about/people/paul-flicek" rel="external">Paul Flicek</a>
-at the EMBL-EBI.
-</p>
+<p>Formal requests such as large scale collaborations, legal issues, etc., are best sent to Rob Finn and/or Andy Yates at EMBL-EBI.</p>
```

Comment:
Links to the people/person pages in EMBL-EBI for Rob Finn and Andy Yates are missing here

Reply:
Good catch, just added them

Reply:
Should these be email addresses (mailTo)?

Reply:
The linked pages contains their e-mail. If we link to that page we won't have to update this again should their e-mail addresses change. They also give some context to who they are which might be useful to whomever wishes to contact them

Reply:
OK Cool